src/pages/index.tsx

```diff
@@ -21,42 +27,26 @@ const Home: NextPage = () => {
         <meta name="description" content="Generated by create-t3-app" />
         <link rel="icon" href="/favicon.png" />
       </Head>
+      <Hero />
 
-      <nav className="flex justify-center gap-4">
-        <Link href="/electricity">Electricity</Link>
-        <Link href="/fuel">Fuel</Link>
-        <Link href="/travel">Travel</Link>
-        <Link passHref href="/auth/login">
-          Log in
-        </Link>
-        <Link href="/register">Register</Link>
-      </nav>
+      <MainBlob 
+        image={WheelchairGuy}
+        headline={"Make an estimate based on your lifestyle."}
+        text={"We'll sum what you consumed in your car, in your flights, to heat your house and give you the total estimate of how your lifestyle impacts the environment."}
```

Comment:
I'd suggest changing `We'll sum what you consumed`... to `We offer you the ability to track your emissions by your various habits, whether that be`...